src/lib/index.jsx

```diff
@@ -135,6 +138,22 @@ class OtpInput extends Component<Props, State> {
   getOtpValue = () =>
     this.props.value ? this.props.value.toString().split('') : [];
 
+  getPlaceholderValue=()=>{
+    const {placeholder,numInputs}= this.props;
+
+    if(!placeholder) return;
```

Comment:
Since placeholder is always going to be a string, having a `typeof placeholder === 'string'` check would prevent conditions including placeholder = 0 from falling through.

Reply:
@apollonian I have made this changes.